static/javascripts/components/profile_groups.jsx

```diff
@@ -0,0 +1,174 @@
+var ProfileGroups = React.createClass({
+    getInitialState: function(){
+        return({
+            currentGroups: [],
```

Comment:
this should be `currentGroups: user_data[0].subscribed_groups` no? Cuz how else are you going to get groups subscribed from the database.